tests/env.earth

```diff
@@ -4,3 +4,14 @@ FROM alpine:3.18
 test:
     ENV MYPATH=hello:$PATH
     RUN echo $MYPATH | grep bin
+
+test-multiple-on-single-env:
+    ENV GOLANG=1.22.2 \
+        GO_VERSION=1.22.2 \
+    	GOOS=linux \
+    	GOARCH=amd64 \
+        GO_DOWNLOAD_SHA256=5901c52b7a78002aeff14a21f93e0f064f74ce1360fce51c6ee68cd471216a17
+
+    RUN env
+    RUN echo GOLANG=$GOLANG
```

Comment:
these tests are not adequate; instead you'll need to do something like
```suggestion
    RUN test "$GOLANG" = "1.22.2"
```

can you additionally rename these values to be something more generic -- using go-specific env names doesn't seem as readable, since at quick glance one might think this test is related to compiling go.

maybe

```
ENV A=alpha \
  B=bravo
```

Reply:
upto u, i m open to it. Just felt it was more relatable to real life

Reply:
`    RUN test "$GOLANG" = "1.22.2"` i didn't realize this is possible . I m curious why it wasn't done for L5

Reply:
done, complete with generic vars